cristin-commons/src/main/java/no/unit/nva/utils/VersioningUtils.java

```diff
@@ -15,4 +18,15 @@ public static String extractVersion(String headerValue) {
         return contentType.getParameter(VERSION);
     }
 
+    /**
+     * Extract the version field value if present from header with key directly from RequestInfo or else null.
+     */
+    public static String extractVersionFromRequestInfo(RequestInfo requestInfo, String headerKey) {
```

Comment:
Can `headerKey` be inlined in the method since it is constant and part of the method name?

Reply:
If we want a versioned post or patch we could pass header Content-Type instead of Accept, so i decided to pass the header key as a parameter